static/sass/components/_author.scss

```diff
@@ -0,0 +1,108 @@
+// Author Box
+// ==========
+
+[data-module] {
+  + [data-module='rst-author'] {
+    border-top: 0;
+    margin-top: size('author');
+    padding-top: 0;
+  }
+}
+
+.author-summary {
+  max-width: 100%;
+}
+
+// crop triangle out of svg for birds page
+.bird-face {
+  float: left;
+  height: size('author');
+  margin-right: size('half-shim');
+  overflow: hidden;
+}
+
+.h-card {
+  [data-module='rst-author'] & {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: size('full-content');
+    padding: 0;
+
+    @include above('full-content') {
+      padding: 0 size('gutter');
+      transform: translateX(negative('half-author'));
+    }
+  }
+}
+
+.bio {
+  clear: both;
+}
+
+.bio-short {
+  padding-left: size('half-author');
+  padding-top: size('pixel'); // hack to get margin to to work
+}
+
+[data-size='face'] {
+  color: color('text');
+  float: left;
+  height: size('author-h');
```

Comment:
@mirisuzanne ah ha! I changed it from author-w to just author because I crop the height to just the square for the birds page (removes triangle).

so I use: 
```
.bird-face {
  height: size('author');
}
```

Thought it would be weird to be:

```
.bird-face {
  height: size('author-width');
}
```
thoughts?